README.md

```diff
@@ -51,65 +50,78 @@ spec:
+I0223 07:33:59.500418 1 nodemaintenance_controller.go:449] evicting pod openshift-network-diagnostics/network-check-source-865d4b5578-n2cxg
+I0223 07:33:59.500790 1 nodemaintenance_controller.go:449] evicting pod openshift-ingress/router-default-7548cf6fb5-rgxrq
+I0223 07:33:59.500944 1 nodemaintenance_controller.go:449] evicting pod openshift-marketplace/12a4cfa0c2be01867daf1d9b7ad7c0ae7a988fd957a2ad6df0d72ff6875lhcx
+I0223 07:33:59.501061 1 nodemaintenance_controller.go:449] evicting pod openshift-marketplace/nmo-downstream-8-8nms7
 ...
 ```
 
 ### Set Maintenance off - Delete the NodeMaintenance CR
 
-To remove maintenance from a node, delete the corresponding `NodeMaintenance` CR:
+To remove maintenance from a node, delete the corresponding `NodeMaintenance` (or `nm` which is a shortName) CR:
 
 ```sh
-$ kubectl delete nodemaintenance nodemaintenance-sample
-
+$ kubectl delete nm nodemaintenance-sample
+nodemaintenance.nodemaintenance.medik8s.io "nodemaintenance-sample" deleted
 $ kubectl logs <nmo-pod-name>
-{"level":"info","ts":1551794725.0018933,"logger":"controller_nodemaintenance","msg":"Reconciling NodeMaintenance","Request.Namespace":"default","Request.Name":"node02"}
-{"level":"info","ts":1551794725.0021605,"logger":"controller_nodemaintenance","msg":"NodeMaintenance Object: default/node02 Deleted ","Request.Namespace":"default","Request.Name":"node02"}
-{"level":"info","ts":1551794725.0022023,"logger":"controller_nodemaintenance","msg":"uncordon Node: node02"}
-
+2022-02-24T14:27:35.332Z INFO controller-runtime.manager.controller.nodemaintenance Reconciling NodeMaintenance {"reconciler group": "nodemaintenance.medik8s.io", "reconciler kind": "NodeMaintenance", "name": "nodemaintenance-sample", "namespace": ""}
+time="2022-02-24T14:27:35Z" level=info msg="Maintenance taints will be removed from node node02"
+time="2022-02-24T14:27:35Z" level=info msg="Applying medik8s.io/drain taint remove on Node: node02"
+...
 ```
 
 ## NodeMaintenance Status
 
-The NodeMaintenance CR can contain the following status fields:
+The `NodeMaintenance` CR can contain the following status fields:
 
 ```yaml
+$ kubectl get nm nodemaintenance-sample -o yaml
 apiVersion: nodemaintenance.medik8s.io/v1beta1
 kind: NodeMaintenance
 metadata:
-  name: nodemaintenance-xyz
+  name: nodemaintenance-sample
 spec:
   nodeName: node02
-  reason: "Test node maintenance"
+  reason: Test node maintenance
```

Comment:
In the old version we use double quotes for all strings.
Here you remove quotes at all, later you use single quotes.
Would be nice to be consistent IMHO

Reply:
I am doing that since this it how it was printed in CLI.
See below example (I don't add or delete quotes, I only take some of the fields in the same output format it is printed). 

➜  Downloads oc get nm nodemaintenance-w-0 -o yaml
apiVersion: nodemaintenance.medik8s.io/v1beta1
kind: NodeMaintenance
metadata:
  creationTimestamp: "2022-02-28T10:48:38Z"
  finalizers:
  - foregroundDeleteNodeMaintenance
  generation: 1
  name: nodemaintenance-w-0
  resourceVersion: "3429673"
  uid: 5b882f87-cd6b-40c4-9920-67dc901796b3
spec:
  nodeName: worker-0
  reason: Test node maintenance
status:
  evictionPods: 5
  lastError: 'error when waiting for pod "router-default-7548cf6fb5-8xj8b" terminating:
    global timeout reached: 30s'
  phase: Succeeded
  totalpods: 19
  
I guess that this result, CR's status, was the same (regarding the quotes) when the documentation was written. Thus, why did we use the old convention/version ? I believe the new version is more close to the actual output.

Reply:
I don't mind changing it back, I am just curious why it was like that in the beginning.

Reply:
ok, I wasn't aware that this is the actual output

Reply:
Alright, so can I have a /lgtm

Reply:
you wanted to address my other comment, not?

Reply:
True... I have forgotten about that, silly me :)